src/components/Header/Header.tsx

```diff
@@ -123,8 +127,15 @@ class Header extends React.Component<any, State>{
                         open={Boolean(this.state.anchorEl)}>
                         <MenuItem onClick={() => { this.props.history.push('/profile'); this.handleClose() }}>Ver Perfil</MenuItem>
                         <MenuItem onClick={() => { this.props.history.push('/'); this.handleClose() }}>Ver Reseñas</MenuItem>
-                        <MenuItem onClick={() => { this.props.history.push('/books'); this.handleClose() }}>Crear Libro</MenuItem>
-                        <MenuItem onClick={() => { this.props.history.push('/authors'); this.handleClose() }}>Crear Autor</MenuItem>
+                        {/*<MenuItem onClick={() => { this.props.history.push('/'); this.handleClose() }}>Ver Reseñas</MenuItem>*/}                        
```

Comment:
Y aca arriba de este yo me dejaria un comment que podamos buscar despues tipo
`{/*TODO: Uncomment when review functionality is done*/}`